bin/console

```diff
@@ -1,29 +1,39 @@
 #!/usr/bin/env php
 <?php
 
-declare(strict_types=1);
-
+use App\Kernel;
 use Symfony\Bundle\FrameworkBundle\Console\Application;
 use Symfony\Component\Console\Input\ArgvInput;
 use Symfony\Component\Debug\Debug;
-
-// if you don't want to setup permissions the proper way, just uncomment the following PHP line
-// read http://symfony.com/doc/current/setup.html#checking-symfony-application-configuration-and-setup
-// for more information
-//umask(0000);
+use Symfony\Component\Dotenv\Dotenv;
 
 set_time_limit(0);
 
 require __DIR__.'/../vendor/autoload.php';
 
+if (!class_exists(Application::class)) {
+    throw new \RuntimeException('You need to add "symfony/framework-bundle" as a Composer dependency.');
+}
+
+if (!isset($_SERVER['APP_ENV'])) {
+    if (!class_exists(Dotenv::class)) {
+        throw new \RuntimeException('APP_ENV environment variable is not defined. You need to define environment variables for configuration or add "symfony/dotenv" as a Composer dependency to load variables from a .env file.');
+    }
+    (new Dotenv())->load(__DIR__.'/../.env');
```

Comment:
Maybe we can use dist files by default? see https://github.com/Sylius/Sylius/pull/9717/files#diff-834e3741acba32abc672bec6be613b2fR23